### DISCUSSION TITLE
Is there a method to import OSM files?

### DISCUSSION THREAD

**Discussion by ip6li** [organicmaps/organicmaps#5108] [Q&A]
I would like to import an OSM file generated by BBBike. Is there a method to import such a file into Organic Maps?

**Comment by biodranik** [comment]
OM generator supports standard OSM input format. Why do you need to import some extract from BBBike? What is your final goal?

**Comment by ip6li** [comment]
Following problem:

* OSM offers on their webpage "Layer=C" which shows trails for bikes.
* BBBike offers an export with that layer.

I would to be able to select "bike layer" in Organic Maps. Layer symbol (button at top left corner) offers topology and underground, only.

**Reply by biodranik** [reply]
There were plans to add hiking and biking routes from OSM on the map. Maybe there were already some related issues for that.
